lib/pluginmanager/util.rb

```diff
@@ -27,7 +32,7 @@ def self.logstash_plugin?(plugin, version = nil)
         end
         return valid
       else
-        $stderr.puts("Plugin #{plugin}" + (version ? " version #{version}" : "") + " does not exist")
+        $stderr.puts("Plugin #{plugin}" + (version ? " version #{version}" : "") + " does not exist") unless error.length != 0
```

Comment:
Replaced with `if errors.empty?` (to only display the plugin does not exist message if no errors were reported)